src/time.rs

```diff
@@ -273,26 +269,40 @@ impl Time {
     /// assert_eq!(d.to_string(), "01:02:20.000123");
     /// ```
     pub fn from_timestamp_with_config(
-        timestamp_second: u32,
+        timestamp: i64,
```

Comment:
Because of the limit of valid input I think the original `u32` input made more sense.

```suggestion
        timestamp: u32,
```

Reply:
working on it